homeassistant/components/unifiprotect/services.yaml

```diff
@@ -84,3 +84,29 @@ set_doorbell_message:
           step: 1
           mode: slider
           unit_of_measurement: minutes
+set_chime_paired_doorbells:
+  name: Set Chime Paired Doorbells
+  description: >
+    Use to set the paired doorbell(s) with a smart chime.
+  fields:
+    entity_id:
```

Comment:
```suggestion
    chime_entity_id:
```

Reply:
This one is confusing as to which entities go in which field if its called `entity_id`

Reply:
Switched it to `device_id` instead to be consistent with the other devices and not require a breaking change down the road.